dev/com.ibm.websphere.appserver.features/visibility/auto/io.openliberty.autoPasswordUtilities1.0.jakarta.feature

```diff
@@ -0,0 +1,14 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=io.openliberty.autoPasswordUtilities1.0.jakarta
+IBM-Provision-Capability: \
+ osgi.identity; filter:="(&(type=osgi.subsystem.feature)(osgi.identity=com.ibm.websphere.appserver.passwordUtilities-1.0))", \
+ osgi.identity; filter:="(&(type=osgi.subsystem.feature)(osgi.identity=com.ibm.websphere.appserver.servlet-5.0))"
+IBM-Install-Policy: when-satisfied
+-features=\
+ com.ibm.websphere.appserver.authData-2.0, \
+ io.openliberty.appSecurity-4.0, \
+ io.openliberty.appserver.connectors-2.0, \
```

Comment:
Previously this only depended on the API feature and not the public feature.  So this should be io.openliberty.jakarta.connectors.

Reply:
Updated to be io.openliberty.jakarta.connectors.